### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by Xdavius** [fastfetch-cli/fastfetch#683]
# General description of bug:

bug on Debian: Logo not show

* What happened:

Using our customization here : https://github.com/Gaming-Linux-FR/GLFfetch
black screen with console
blackscreen with kitty
no logo with terminator

* What should happen:

should be like the demo in the project

* Did it work in an older version: 
* Where did you get the binary: 

tested deb-get / directly .deb github

* Does this issue still occurs in [the latest dev build](https://github.com/fastfetch-cli/fastfetch/actions/)?

Sorry for no more explainations. Works on archlinux, but not in debian. I supose I missed something but don't know what...
your help will be appréciated.

regards

Davius

**Comment by CarterLi** [comment]
What does black screen mean? Did it crash? You may try gdb to get some stack trace. I'm sorry but the information provided is not enough for me to identify what the problem is.

**Comment by CarterLi** [comment]
According to <https://github.com/gnome-terminator/terminator/pull/770>, in order to enable image support for terminator, you need a custom build of [Gnome VTE](https://gitlab.gnome.org/GNOME/vte) with sixel support.

**Comment by Xdavius** [comment]
Hi @CarterLi 

I'm still searching exactly what is the problem. The crash I discribed is that the terminal do just nothing. No fetch att all and need to ctrl+c.

I was able to run our custom config. But I've done too much things to be clear.

1- seems I need to recompile fastfetch
2- libmagick++* and libmagick++dev seems to be needed but not sure.

I will test and come back with more info but seems to be a dépendancies problem and maybe a compilation problem with the.deb you provide.

Thank you for reading.

Regards

Xdavius
